### ISSUE TITLE
Possible Anthropic compatability with upgrading any-llm dependency

### ISSUE THREAD

**Issue by erkandem** [wagtail/django-ai-core#12]
### Problem statement

While I have read, that the package is modelled for OpenAI API compatability might not be far away:
Particularly image alt text generation and content feedback generation is failing as of the combo of:

```
llm==0.31
wagtail-ai==3.1.0
django-ai-core==0.1.5
anthropic==0.100.0
```

### Proposed solution

Version 1.8.6 might be the key, which I would happy if you could consider it.
But I'm aware, it might not be as simple as upgrading the referenced version if pyproject.toml bc it might break other areas of django-ai-core.
https://github.com/mozilla-ai/any-llm/releases/tag/1.8.6

### Additional context

Before 1.8.6 any llm used to raise an error bc it was unsupported.
https://github.com/mozilla-ai/any-llm/pull/831/changes#diff-cfa023796f9099c5df0e90666971ca48fed96b6d50842d7184c3daac022eacbfL349
